examples/vic-gov-au/assets/ripple-icon/icon_circle_arrow_right.js

```diff
@@ -0,0 +1,13 @@
+import Icon from '../../../../packages/components/Atoms/Icon/Icon'
```

Comment:
In e.g. vic-gov-au, the import path will be '@dpc-sdp/ripple-icon'. I just use relative path here for example